string status
int32[] value
